ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh

```diff
@@ -67,6 +76,7 @@ platform:
   nutanix:${RHCOS_PATCH}
     apiVIP: ${API_VIP}
     ingressVIP: ${INGRESS_VIP}
+    ${LB_TYPE_DEF}
```

Comment:
One question: when it uses external lb, do we still need specific apiVIP and ingressVIP in install-config.yaml? For vsphere, seems no need to set VIPs if using external lb.

Reply:
yes, this is a new feature from network team, @zhaozhanqi also added a similar workflow on vSphere here https://github.com/openshift/release/pull/47676#discussion_r1540365407

Reply:
@jinyunma the little difference is:
1.  if without apivip/ingressvip in install-config, it's should be UPI,  the cluster do not have openshift-nutanix-infra project I guess
2. for this way specified LB type: userManaged and apiVIP/ingressVIP. which only coredns will be running on openshift-nutanix-infra. this should be IPI

Thanks to @sgaoshang helping implement this workflow...